pubsub.go

```diff
@@ -135,12 +135,17 @@ type PubSubRouter interface {
 
 type Message struct {
 	*pb.Message
+	senderID peer.ID
```

Comment:
this is available in the `RPC` struct and will be exposed to routers in the refactoring in #199.
It is also visible to validators as it is passed in the validation function.